### DISCUSSION TITLE
Setting up an ssh key "error" | convert private to open

### DISCUSSION THREAD

**Discussion by CodyDale2000** [community/community#136130] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hi, I just started the rock the code program apart of the power education and i am having trouble setting up a new ssh key within git hub because my computer defaults to a private key and git hub wants it in a open pubilc ssh key format?? how do i make it so it works?? any thoughts??

**Comment by interkenny** [comment]
Docs:
- Create a new SSH key
https://docs.github.com/en/authentication/connecting-to-github-with-ssh/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent

- Adding a new SSH key to your  account
https://docs.github.com/en/authentication/connecting-to-github-with-ssh/adding-a-new-ssh-key-to-your-github-account
